force-app/main/default/objects/Contact/recordTypes/Speaker.recordType-meta.xml

```diff
@@ -4,6 +4,44 @@
     <active>true</active>
```

Comment:
@bmtemakina  I believe this change in Speaker record type is related to Contact object, is that correct ? if so please remove this metadata form this PR and create another PR for this change.